interop-testing/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
             project(':grpc-testing'),
             libraries.junit,
             libraries.mockito,
+            libraries.netty_tcnative,
```

Comment:
@nmittler, this surprises me, mainly because I thought we already had the netty_tcnative dependency in place here. It seems we are including tcnative in the binaries (via the `distributions` section below) but was using Jetty ALPN during `gradle build`. Do you remember if we did not use tcnative here before because of concerns about needing OpenSSL installed, or was it something different?